### PR TITLE
Doc7359 blobs

### PR DESCRIPTION
**Pull request by barbnewrelic**
New logs blobs docs for DOC-7359

### CODE REVIEW

**Comment by barbnewrelic** [comment]
@mculmone  At your convenience, would you please review this PR? This is for LOGGING-5985 (parsing FAQ) and LOGGING-5917 (blobs). Feel free to ask other Logs SMEs to review as well. Thanks!

File: src/content/docs/logs/log-management/log-api/log-event-data.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+## Data storage [#events]
+
+Log records are stored by default in the `Log` event type. You can create additional event types by defining a custom data partition in Logs. The resulting types will always be prefaced with `Log_`. For detailed information, see our [data partitions documentation](/docs/logs/log-management/ui-data/data-partitions/).
+
+**Attributes:**
+
+* Maximum attributes: 255
+* Name: maximum 255 characters
+* Searchable storage: First 4096 characters
```
Comment (mculmone) [inline src/content/docs/logs/log-management/log-api/log-event-data.mdx:25]: Can we have it as "Searchable value"? And use bytes not chars

* Searchable value: The first 4096 bytes of data stored in an attribute can be queried directly in the UI

File: src/content/docs/logs/log-management/log-api/log-event-data.mdx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+**Blob storage:**
```
Comment (mculmone) [inline src/content/docs/logs/log-management/log-api/log-event-data.mdx:27]: ** Additional storage:**

**Comment by barbnewrelic** [comment]
Logs team needs these docs to support some upcoming marketing info, so i'm going to merge this now.

**Pull request merged**